README.md

```diff
@@ -2,6 +2,20 @@
 
 This repository is used to download a set of components that are consistent with each other in terms of compatibility and versioning. This repository has all the components and scripts to run each machine (BMD, BAS, BSD, EMS).
 
+## Developing vxsuite through kiosk-browser
+If you are developing a change in [vxsuite](https://github.com/votingworks/vxsuite) and want to test it through kiosk-browser to mimic production, follow the steps below. Only do this *after* you have run the `setup-dev` script in vxsuite, which will install node and other dependencies.
+
+```sh
+make checkout
+make build-kiosk-browser
+# If you are using debian bullseye add your user to the lpadmin group
+sudo usermod -aG lpadmin <USERNAME> # Replace <USERNAME> with your linux username
```

Comment:
if you just make this `$USER` instead of `<USERNAME>` then it will substitute the user's name automatically.